Jenkinsfile

```diff
@@ -1,166 +1,172 @@
+      	    
+      	    		sh "mvn -Djacoco.skip=true -DjavaDocPhase=none ${customParams} clean package -f pom.xml"
+	       		}         
+	     	}
+	    }
+	    
+	    stage('Download Capella') {
+        	steps {
+        		script {
+	        		def capellaURL = capella.getDownloadURL("${BUILD_KEY}", 'linux', '')
+	        		
+	        		sh "curl -k -o capella.zip ${capellaURL}"
+					sh "unzip -q capella.zip"
 
-  post {
-    always {
-       archiveArtifacts artifacts: '**/*.log, **/Cybersecurity*.zip, **/capella-darc*.zip'
-    }
-  }
+	       		}         
+	     	}
+	    }
+	    
+	    stage('Adapt Capella to DARC') {
+	        steps {
+	            //Install Cybersecurity on Capella
+	            sh "ls -lat ."
+	            sh "ls -lat ${WORKSPACE}/releng/org.polarsys.capella.cybersecurity.site/target/repository"
+	            sh "chmod 755 ./capella/capella"
+	            sh "./capella/capella -application org.eclipse.equinox.p2.director -repository file:/${WORKSPACE}/releng/org.polarsys.capella.cybersecurity.site/target/repository -installIU org.polarsys.capella.cybersecurity.feature.feature.group -noSplash"            
+	            
+	            //Adapt eclipse.ini config.ini and other things
+	            sh "cp -rp capella capella-darc"
+	            sh "cat capella-darc/capella.ini"
+	            sh "cat capella-darc/configuration/config.ini"
+	            sh "sed -i \"s,eclipse.product=org.polarsys.capella.rcp.product,eclipse.product=org.polarsys.capella.cybersecurity.rcp.product,g\" capella-darc/configuration/config.ini"
+	            sh "sed -i 's,osgi.splashPath=platform\\\\:/base/plugins/org.polarsys.capella.core.platform.sirius.ui.perspective,osgi.splashPath=platform\\\\:/base/plugins/org.polarsys.capella.cybersecurity.rcp,g' capella-darc/configuration/config.ini"
+	            sh "cat capella-darc/configuration/config.ini"
+	            sh '''
+	                # Identify darc name
+	                addon_local_dropins_name=`ls ${WORKSPACE}/releng/org.polarsys.capella.cybersecurity.site/target | grep "Cybersecurity-dropins" | cut -d"-" -f3 | sed "s/.zip//"`
+	                # Build the product zip
+	                DARC_BUILD="capella-darc-${addon_local_dropins_name}-win32-win32-x86_64.zip"
+	                zip -q -r $DARC_BUILD capella-darc
+	            '''
+	        }
+	    }
+
+		stage('Deploy to Nightly') {
+      		steps {
+				script {
+		    		def deploymentDirName = 
+		    			(github.isPullRequest() ? "${BUILD_KEY}-${BRANCH_NAME}-${BUILD_ID}" : "${BRANCH_NAME}-${BUILD_ID}")
+		    			.replaceAll('/','-')		
+					
+					deployer.addonNightlyDropins("${WORKSPACE}/releng/org.polarsys.capella.cybersecurity.site/target/*-dropins-*.zip", deploymentDirName)
+					deployer.addonNightlyUpdateSite("${WORKSPACE}/releng/org.polarsys.capella.cybersecurity.site/target/*-updateSite-*.zip", deploymentDirName)					
+					deployer.addonNightlyUpdateSite("${WORKSPACE}/capella-darc-*-win32-win32-x86_64.zip", deploymentDirName)
```

Comment:
I there is an issue here, since this publishes the product in the [update site directory](https://download.eclipse.org/capella/addons/cybersecurity/updates/nightly/master-PR-11-6/)

Previously the product was in the [products directory](https://download.eclipse.org/capella/addons/cybersecurity/products/nightly/) 

I think we must add a function in the pipeline factory, similar to `deployer.addonNightlyDropins` called `deployer.addonProduct` that does this